### PR TITLE
Updated readme github buttons

### PR DESCRIPTION
**Pull request by swathiswaminathan**
Added a 'open in cloudshell' feature into the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,22 @@ Once the container image has been built and the necessary attestations have been
 * A Google Cloud account and project is required for this demo. The project must have the proper quota to run a Kubernetes Engine cluster with at least 2 vCPUs and 7.5GB of RAM. How to check your account's quota is documented here: [quotas][1].
 * Note: the Binary Authorization and Container Analysis APIs are currently in `beta`. [Read more][9] about the support level and SLAs given to APIs designated as `beta` before using these features in a production environment.
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```
Comment (emailbob) [inline README.md]: `[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

**Comment by emailbob** [comment]
This project already has a Google Cloud Shell button. We need to keep one of them.

**Comment by chrislovecnm** [comment]
Test this please

**Comment by chrislovecnm** [comment]
Please test this

**Comment by chrislovecnm** [comment]
@swathiswaminathan is this PR passing?  What is going on with this Jenkins Job?

**Comment by emailbob** [comment]
test this please

**Pull request merged**